stickytimelineview/src/main/res/values/attrs.xml

```diff
@@ -14,5 +14,9 @@
         <attr name="sectionSubTitleTextSize" format="dimension" />
         <attr name="isSticky" format="boolean" />
         <attr name="customDotDrawable" format="reference" />
+        <attr name="mode" format="enum">
```

Comment:
In my opinion, It would be better if the name was more detailed than `mode`. In general, in case of linear layout or recycler, the name is used as `orientation`, so it is better to use a name with` orientation`. (ex: timeLineOrientation)
What do you think about this?

Reply:
you're right,, but i think mode is better than orientation,
because someone make the unique sticky timeline mode , not just horizontal and vertical
for example, someone makes 2lines skticky time line ... etc
if it's name is orientation, it will not have all meaning,
so i named this property as "mode"

Reply:
Yes it seems to be a good approach :)